### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sprigunn**
Typos and additional license to sync up with http://homebrewery.naturalcrit.com.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,14 +20,16 @@ Homebrewery is open source, so feel free to clone it, tinker with it, or run you
...
 
 ### changelog
 
-You can check out the changelog [here](https://github.com/stolksdorf/homebrewery/blob/master/changelog.md)
+You can check out the changelog [here](https://github.com/stolksdorf/homebrewery/blob/master/changelog.md).
 
 ### license
 
-This project is licensed under [MIT](./license)
+This project is licensed under [MIT](./license). Which means you are free to use The Homebrewery is any way that you want, except for claiming that you made it yourself.
```
Comment (ericscheid) [inline README.md]: "free to use The Homebrewery is any way" ⇒ "free to use The Homebrewery **in** any way"

Comment (sprigunn) [reply README.md]: Thanks for picking that up.

Comment (Rae2che5) [reply README.md]: Happy to merge once this is fixed up.

Comment (calculuschild) [reply README.md]: I got it. Going to merge this up.

**Pull request merged**